.github/workflows/android_build.yml

```diff
@@ -21,4 +21,4 @@ jobs:
       - name: Wrapper permissions
         run: chmod +x ./gradlew
       - name: Build
-        run: ./gradlew assemble -PartifactoryUsername=${{secrets.ARTIFACTORY_USERNAME}} -PartifactoryPassword=${{secrets.ARTIFACTORY_PASSWORD}} -PtomtomApiKey=${{secrets.TOMTOM_API_KEY}}
+        run: ./gradlew assemble -PrepositoryUserName=${{secrets.ARTIFACTORY_USERNAME}} -PrepositoryIdentityToken=${{secrets.ARTIFACTORY_PASSWORD}} -PtomtomApiKey=${{secrets.TOMTOM_API_KEY}}
```

Comment:
Shouldn't we use `User Name` when we are referencing to a real person (e.g. Adrian Glasnapp), but when we talk about authentication it is `Username` (e.g. adrianglasnapp-tomtom) ?

Reply:
you are right, we have to change it again in the developer portal